Makefile

```diff
@@ -246,8 +224,7 @@ CMAKE_FILES+= \
 	tests/ctests/CMakeLists.txt \
 	tests/cpptests/CMakeLists.txt \
 	tests/cpptests/redismock/CMakeLists.txt \
-	tests/pytests/CMakeLists.txt \
-	tests/c_utils/CMakeLists.txt
+	tests/pytests/CMakeLists.txt
```

Comment:
not your doing, but do we really need pytsts/CMakeLists.txt?

Reply:
Actually, it seems like none of this is necessary (some leftovers probably...) - removing it all